### ISSUE TITLE
Not support QtCurve opacity

### ISSUE THREAD

**Issue by varlesh** [QupZilla/qupzilla#1948]
With new release 2.0 Qupzilla not suppor window background opacity on QtCurve.
PS: Other Qt5-apps support this option (Dolphin, QBittorrent).
![screenshot_20160416_010336](https://cloud.githubusercontent.com/assets/8083855/14576143/6d764c2a-036f-11e6-8646-986732ff3d95.png)
Please add  support)

**Comment by varlesh** [comment]
Only menus support this option:
![screenshot_20160416_011740](https://cloud.githubusercontent.com/assets/8083855/14576367/28c65528-0371-11e6-9274-711e160836cc.png)

**Comment by nowrep** [comment]
I guess it should work also with other windows as site info, preferences, adblock dialog, ...

**Comment by varlesh** [comment]
Yes, other windows (Adblock, site info, prefereces and etc...) support opacity. But main window not support :(

**Comment by nowrep** [comment]
It seems to be because QtWebEngine, so QtCurve needs to try some hack to get it working (if possible).
